daemon/daemon_windows.go

```diff
@@ -238,6 +240,29 @@ func checkSystem() error {
 		return fmt.Errorf("Failed to load vmcompute.dll. Ensure that the Containers role is installed.")
```

Comment:
This should probably be updated to `feature` too

Reply:
Since I'd like to backport this fix to 17.06, I think this should be done in a followup to prevent changing error messages in an already released version. Just in case.

Reply:
Makes sense - can you open a follow-up PR to change this?